commands/diff.go

```diff
@@ -4,13 +4,15 @@ import (
 	"io"
 	"os"
 
+	"github.com/10gen/realm-cli/utils/telemetry"
+
```

Comment:
this shouldn't be in its own import group, and as a driveby fix could you move `mitchellh/cli` into its own 3rd group?